README.md

```diff
@@ -1,79 +1,142 @@
-# nginx-k8s-loadbalancer
+<div>
+    <span>
+<img src="nkl-logo.svg" width="124" />
+</span>    
+<span>
+<h2>nginx-k8s-loadbalancer</h2>
+</span>
+</div>
 
-## Welcome to the Nginx Kubernetes Load Balancer project !
+The NGINX K8s Loadbalancer, or _NKL_, is a Kubernetes controller that provides TCP load balancing external to a Kubernetes cluster running on-premise.
 
-<br/>
+## Requirements
 
-This repo contains source code and documents for a new Kubernetes Controller, that provides TCP load balancing external to a Kubernetes Cluster running On Premises.  
+### Who needs NKL?
 
-<br/>
+- [ ] If you find yourself living in a world where Kubernetes is running on-premise instead of a cloud provider, you might need NKL.
+- [ ] If you want exceptional, best-in-class load-balancing for your Kubernetes applications, you might need NKL.
+- [ ] If you want the ability to manage your load-balancing configuration with the same tools you use to manage your Kubernetes cluster, you might need NKL.
 
->>**This is a replacement for a Cloud Providers "Service Type Loadbalancer", that is missing from On Premises Kubernetes Clusters.**
+### Why NKL?
 
-<br/>
+NKL provides a simple, easy-to-manage way to automate load balancing for your Kubernetes applications by leveraging NGINX Plus hosts running outside your cluster.
 
-## Overview
+NKL installs easily, has a small footprint, and is easy to configure and manage.
 
-- Create a new K8s Controller, that will monitor specified k8s Services, and then send API calls to an external Nginx Plus server to manage Nginx Upstream servers automatically.  
-- This will `synchronize` the K8s Service Endpoint list, with the Nginx LB server's Upstream server list.  
-- The primary use case is for tracking the NodePort IP:Port definitions for the Nginx Ingress Controller's `nginx-ingress Service`.  
-- With the Nginx Plus Server located external to the K8s cluster, this new controller LB function would provide an alternative TCP "Load Balancer Service" for On Premises K8s clusters, which do not have access to a Cloud providers "Service Type LoadBalancer".
-- Make the solution a native Kubernetes Component, running, configured and managed with standard K8s commands.
+? {{review for embetterment}}: NKL does not require any specific domain knowledge for configuration, though you will have to understand NGINX configuration to get the most out of this solution. There is thorough documentation available about these specifics in the `docs/` directory.
```

Comment:
I guess my question is which domain you're talking about?  Kubernetes? NEtworking? Load balancing? NGINX?

Reply:
What I am trying to say is that NKL has no domain model of it's own to learn; that you need only understand how to configure NGINX Plus.